src/styles.scss

```diff
@@ -1,4 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
 
 html, body { height: 100%; }
-body { font-family: Roboto, "Helvetica Neue", sans-serif; }
+body {
+  font-family: Roboto, "Helvetica Neue", sans-serif;
+
+  section {
```

Comment:
Are u sure that we need to override global styles for it?

Reply:
Angular Material paste this by default